File: AtCoder/abc209/A.swift
private func readLn() -> String {
	return readLine() ?? ""
}

private func readLns() -> [String] {
	return readLn().split(separator: " ").map{ String($0) }
}

private func readInts() -> [Int] {
	return readLns().compactMap{ Int($0) }
}

let ab = readInts()
var ans = ab[1] - ab[0] + 1
if ans < 0 { ans = 0 }
print(ans)

File: AtCoder/abc209/A.kt
private fun readLn() = readLine() ?: ""
private fun readLns() = readLn().split(" ")
private fun readInts() = readLns().map { it.toInt() }

fun main() {
    val (a, b) = readInts()
    var ans = b - a + 1
    if (ans < 0) ans = 0
    print(ans)
}